packages/components/ng-ovh-sso-auth-modal-plugin/package.json

```diff
@@ -47,7 +47,9 @@
     "@ovh-ux/ng-translate-async-loader": "^2.1.2",
     "angular": "^1.3.x",
     "angular-translate": "^2.18.1",
-    "angular-ui-bootstrap": "1.3.3",
-    "jquery": "^2.1.3"
+    "angular-ui-bootstrap": "1.3.3"
+  },
+  "optionalDependencies": {
```

Comment:
I think `optionalDependencies` are dependencies that are not required to run, but used in your code. 
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#optionaldependencies

Here `whatwg-fetch` is not imported, so not used; it's a _recommended_ dependencies.

If you use `optionnal` you should support the case when dependency is not installed (`npm install --no-optional` or `yarn install --ignore-optional`)

Reply:
Don't know about that, the guy who opened the first PR seems to know what he is doing: https://github.com/ovh/manager/pull/2852/files#diff-6800f225d7d13b98b4781559c10c478d3f4fe3bd2c4ade35ca643f2aa9a8ea6fR51

Reply:
Seen together, dependency is removed, thanks.